platform/android/CHANGELOG.md

```diff
@@ -3,23 +3,33 @@
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
 ## master
-- Enable variable label placement when `text-allow-overlap` property is set to true [#15354](https://github.com/mapbox/mapbox-gl-native/pull/15354).
-- Introduce `text-writing-mode` layout property for symbol layer [#14932](https://github.com/mapbox/mapbox-gl-native/pull/14932). The `text-writing-mode` layout property allows control over symbol's preferred writing mode. The new property value is an array, whose values are enumeration values from a ( `horizontal` | `vertical` ) set.
+
+## 8.3.0-alpha.3 - August 15, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.3.0-alpha.2...android-v8.3.0-alpha.3) since [Mapbox Maps SDK for Android v8.3.0-alpha.2](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0-alpha.2):
+
+### Major changes
+ - Bumped gestures library version to `v0.5.0` which introduces a custom scale gesture detector implementation. The library doesn't rely on the compat gesture detector anymore. This is a softly breaking changing because the underlying scale gesture detector reference has been removed. This brings a set of minor improvements and no functionality is missing in comparison to the previous implementation. [#15136](https://github.com/mapbox/mapbox-gl-native/pull/15136)
 
 ### Features
+ - Introduce `text-writing-mode` layout property for symbol layer. The `text-writing-mode` layout property allows control over symbol's preferred writing mode. The new property value is an array, whose values are enumeration values from a ( `horizontal` | `vertical` ) set. [#14932](https://github.com/mapbox/mapbox-gl-native/pull/14932)
```

Comment:
The `text-writing-mode` layout property allows control whether a symbol should be placed horizontally or vertically.

(this entry is maybe less accurate, but it better exemplifies the purpose of the style spec property)